.github/workflows/runtime_build.yml

```diff
@@ -14,6 +14,11 @@ jobs:
   build_and_lint:
     name: yarn build and lint
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # yml is dumb. update the --total arg to yarn build if you change the number of workers
+        worker_id: [0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19]
```

Comment:
Curious why 20 workers. Does this match the number of bundles?

Reply:
I preserved our current circleci setup, where we have 20 workers for stable and 20 workers for experimental. That's the same as what this config in GH expands to